core/frontend/src/components/kraken/BackAlleyTab.vue

```diff
@@ -30,50 +30,136 @@
+              <span
+                v-if="extension_companies.length > 0"
               >
-                <template #default="{ item }">
-                  <v-card-text>
+                <v-card-subtitle
+                  class="pt-4 pb-0 filter-category-title"
+                >
+                  COMPANIES
+                </v-card-subtitle>
+                <div
+                  v-if="extension_companies.length <= 8"
+                  class="pt-1"
+                >
+                  <v-card-text
+                    v-for="company in extension_companies"
+                    :key="company"
+                    class="mt-0 pt-1"
+                    style="height: 30px !important;"
+                  >
                     <v-checkbox
-                      v-model="selected_companies"
-                      :label="item"
-                      :value="item"
+                      v-model="filter_by_selected_companies"
+                      :label="company"
+                      :value="company"
                       class="pa-0 pl-3 ma-0"
                     />
                   </v-card-text>
-                </template>
-              </v-virtual-scroll>
-              <v-divider />
+                </div>
+                <v-virtual-scroll
+                  v-if="extension_companies.length > 8"
+                  :items="extension_companies"
+                  item-height="30"
+                  height="240"
+                >
+                  <template #default="{ item }">
+                    <v-card-text class="pt-2">
+                      <v-checkbox
+                        v-model="filter_by_selected_companies"
+                        :label="item"
+                        :value="item"
+                        class="pa-0 pl-3 ma-0"
+                      />
+                    </v-card-text>
+                  </template>
+                </v-virtual-scroll>
+              </span>
+              <span
+                v-if="extension_types.length > 0"
+              >
+                <v-card-subtitle
+                  class="pt-4 pb-0 filter-category-title"
+                >
+                  CATEGORIES
+                </v-card-subtitle>
+                <div
+                  v-if="extension_types.length <= 8"
+                  class="pt-1"
+                >
+                  <v-card-text
+                    v-for="category in extension_types"
+                    :key="category"
+                    class="mt-0 pt-1"
+                    style="height: 30px !important;"
+                  >
+                    <v-checkbox
+                      v-model="filter_by_selected_types"
+                      :label="category"
+                      :value="category"
+                      class="pa-0 pl-3 ma-0"
+                    />
+                  </v-card-text>
+                </div>
+                <v-virtual-scroll
+                  v-if="extension_types.length > 8"
```

Comment:
```suggestion
                  v-else
```

Reply:
Great, forgot to change that when added the span wrapper, added!